Makefile

```diff
@@ -1,17 +1,15 @@
+all: requirements quality test
+
 clean:
 	coverage erase
 	find . -name '*.pyc' -delete
 
 quality:
-	pep8 --config=.pep8 organizations
-	pylint --rcfile=pylintrc organizations
+	tox -e quality
 
 requirements:
 	pip install -e .    # Install this package and its dependencies
 	pip install -r test-requirements.txt
 
 test:
-	django-admin.py test organizations --settings=test_settings --with-coverage --cover-package=organizations
-	coverage report
-
-.PHONY: clean, quality, requirements, test
+	tox
```

Comment:
That's an interesting idea, but I think I will hold off on it. We haven't been doing this in general on the upgrades, and these tests in particular are very fast to run (furthermore, it's an area of code we rarely touch). Thank you though for the suggestion, and the review!